pkg/features/kube_features.go

```diff
@@ -196,6 +196,11 @@ const (
 	// Requires careful consideration as it may lead to security issues.
 	// Deprecated: planned to be removed in 0.17
 	MultiKueueAllowInsecureKubeconfigs featuregate.Feature = "MultiKueueAllowInsecureKubeconfigs"
+
+	// owner: @pbundyra
+	//
+	// Enables reclaimable pods counting towards quota.
+	ReclaimablePods featuregate.Feature = "ReclaimablePods"
```

Comment:
Please also adjust the KEP: https://github.com/kubernetes-sigs/kueue/tree/main/keps/78-dynamically-reclaiming-resources

Reply:
Done

Reply:
Done

Reply:
Done